load_dumps.php

```diff
@@ -0,0 +1,25 @@
+<?php
+require('app/config.php');
```

Comment:
This script was supposed to run separately on command line. Currently it is not called from the context of framework.